Update wiki media guidelines (#1919)
hashicorp/design-system: wiki/Website-Media.md

1. Install the desktop app for your specific operating system.
2. Open the application settings and ensure that the settings match the recommended optimization settings:
    1. In the quality tab, enable "Lossy minification"
    2. Set the image qualities for JPG, PNG, and GIF files to 85–90%. This results in a high-quality, high-resolution image while removing unnecessary metadata and compressing the file size for the web.
3. Add images either using the plus icon in the lower left corner or by dragging and dropping files from your file browser into the window.

Once added to the queue, ImageOptim will automatically optimize and save the assets. **Warning:** This will overwrite the original files.

## Adding and optimizing video assets

Providing video examples in documentation can be helpful for explaining complex concepts in a step-by-step manner, but these types of assets require a bit more work to create and subsequently include in markdown.

_Video assets (MP4, MOV, etc.) aren’t directly supported by markdown. For more details on how to include them, see the [markdown documentation](/wiki/Website-Markdown.md)._

> These steps focus on _rendering_ a video asset and are specific to the workflow between Adobe After Effects, Premiere, and Adobe Media Encoder. You will need to seek approval from your manager and request a license from IT to gain access to these applications. More complex examples (creating a custom UI in After Effects, more in-depth editing, etc) are beyond the scope of this guide.

Reviewer: ```suggestion
These steps are specific to the workflow between Adobe After Effects, Premiere, and Adobe Media Encoder. You will need to seek approval from your manager and request a license from IT to gain access to these applications. More complex examples (creating a custom UI in After Effects, more in-depth editing, etc) are beyond the scope of this guide.
```

  - is it an image that resizes when the viewport changes? speak with the developer and decide what is the right size to use

If you have any doubts or problems exporting the files, speak with one of the designers of the HDS team.

## Optimizing images

> Prior to adding new images to the website the assets should be optimized to reduce their file size and increase our web performance. Figma's export feature doesn't allow for much fine-grained control regarding image quality and optimizing for speed on the web. [ImageOptim](https://imageoptim.com) works great, is free, easy to use, and with the correct settings can result in a more than _50% reduction_ in file size per image.

Reviewer: ```suggestion
Prior to adding new images to the website, the assets should be optimized to reduce file sizes and increase web performance. Figma’s export feature doesn’t allow for fine-grained control regarding image quality and optimization for the web. [ImageOptim](https://imageoptim.com) is free, works great, and is easy to use. And with the correct settings, it can result in a more than _50% reduction_ in file size per image.
```

1. Install the desktop app for your specific operating system.

> 2. Open the application settings; generally most of the default settings work out of the box, but for reference:

Reviewer: I think that's clearer vs. mentioning the default settings

>     1. In the quality tab, enable `Lossy minification`

Reviewer: ```suggestion
    1. In the quality tab, enable “Lossy minification”
```

>     2. Set the image qualities for .jpg, .png, and .gif to 85-90%. This results in a high quality, high resolution image (using the guidelines outlined above), while removing unnecessary metadata and compressing the filesize to something more managable for the web.

Reviewer: ```suggestion
    2. Set the image qualities for JPG, PNG, and GIF files to 85–90%. This results in a high-quality, high-resolution image while removing unnecessary metadata and compressing the file size for the web.
```

> 3. Within the app, either add images to the queue with the plus icon in the lower left corner, or simply drag and drop files from your file browser into the window.

Reviewer: ```suggestion
3. Within the app, add images either using the plus icon in the lower left corner or by dragging and dropping files from your file browser into the window.
```

Reviewer: I'm thinking THAT this part is probably understood so could be deleted: "Within the app"

> Once added to the queue assets will be optimized automatically and ImageOptim will automatically save the compressed assets to the same location in the filesystem. **Warning:** This does overwrite the original file.

Reviewer: ```suggestion
Once added to the queue, ImageOptim will automatically optimize and save the assets. **Warning:** This will overwrite the original files.
```

> Once added to the queue assets will be optimized automatically and rather than exporting fundamentally new assets, ImageOptim automatically saves compressed assets to the same location in the filesystem. **Warning:** This does overwrite the original file.

Reviewer 2: ```suggestion
Once added to the queue assets will be optimized automatically and ImageOptim will automatically save the compressed assets to the same location in the filesystem. **Warning:** This does overwrite the original file.
```

## Adding and optimizing video assets

> Including video examples and prototypes in documentation can help to express complex concepts in a step-by-step manner, but these types of assets require a bit more work to create and subsequently include in markdown. 

Reviewer: ```suggestion
Providing video examples in documentation can be helpful for explaining complex concepts in a step-by-step manner, but these types of assets require a bit more work to create and subsequently include in markdown.
```

_Video assets (.mp4, .mov, etc) aren't directly supported by markdown, for more details on how to include them in documentation view the [markdown documentation](/wiki/Website-Markdown.md)._

> These steps are specific to using either Adobe After Effects or Premiere along with Adobe Media Encoder for rendering. These applications are not available by default; you will need to seek approval from your manager and request a license from IT.

Reviewer: ```suggestion
These steps are specific to Adobe After Effects and Premiere, with Adobe Media Encoder used for rendering. You will need to seek approval from your manager and request a license from IT to gain access to these applications.
```

These steps focus on _rendering_ a video asset for use on the web using Adobe Media Encoder. More complex examples (creating a custom UI in After Effects, more in-depth editing, etc) are beyond the scope of this guide.

> 

Reviewer: These two paragraphs need some massaging:

```
These steps are specific to Adobe After Effects and Premiere, with Adobe Media Encoder used for rendering. You will need to seek approval from your manager and request a license from IT to gain access to these applications.

These steps focus on _rendering_ a video asset for use on the web using Adobe Media Encoder. More complex examples (creating a custom UI in After Effects, more in-depth editing, etc) are beyond the scope of this guide.
```

Reviewer: They're too repetitive as is.

> These steps are specific to using either Adobe After Effects or Premiere along with Adobe Media Encoder for rendering. These applications are available by default and you will need to seek approval from your manager and request a license from IT.

Reviewer 2: ```suggestion
These steps are specific to using either Adobe After Effects or Premiere along with Adobe Media Encoder for rendering. These applications are not available by default: you will need to seek approval from your manager and request a license from IT.
```

Author: Shoot good catch on this 🤦

> These steps are focused on _rendering_ a video asset for use on the web using Adobe Media Encoder. More complex examples (creating a custom UI in After Effects, more in-depth editing, etc) are beyond the scope of this guide.

Reviewer: ```suggestion
These steps focus on _rendering_ a video asset for use on the web using Adobe Media Encoder. More complex examples (creating a custom UI in After Effects, more in-depth editing, etc) are beyond the scope of this guide.
```

> 1. Create your source video asset; the most straightforward method is to record your screen with Quicktime's native screen recording feature either showing an in-browser example, or Figma prototype.

Reviewer 2: ```suggestion
1. Create your source video asset; the most straightforward method is to [record your screen with Quicktime's native screen recording feature](https://support.apple.com/en-gb/guide/quicktime-player/qtp97b08e666/mac) either showing an in-browser example, or Figma prototype.
```

> 1. Create your source video asset; the most straightforward method is to [record your screen with Quicktime's native screen recording feature](https://support.apple.com/en-gb/guide/quicktime-player/qtp97b08e666/mac) either showing an in-browser example, or Figma prototype.

Reviewer: ```suggestion
1. Create your source video, for example, of an in-browser example or a Figma prototype; the most straightforward method is to [record your screen using Quicktime’s native screen recording feature](https://support.apple.com/en-gb/guide/quicktime-player/qtp97b08e666/mac).
```

2. After launching Media Encoder, add your asset to the render queue.

>     1. In After Effects this can be done through the "Composition" menu and selecting "Add to Media Encoder Render Queue…" (shortcut `Alt` + `Cmd` + `m`).

Reviewer: ```suggestion
    1. In After Effects this can be done through the "Composition" menu by selecting "Add to Media Encoder Render Queue…" (shortcut `Alt` + `Cmd` + `m`).
```

>     2. In Premiere this can be done through the "File" menu, Export -> Send to Adobe Media Encoder (shortcut `Cmd` + `Shift` + `m`) .

Reviewer: ```suggestion
    2. In Premiere, use the "File" menu, Choosing "Export -> Send to Adobe Media Encoder" (Cmd + Shift + m) .
```

>     3. If adding directly to Media Encoder, click the `+` button in the Queue panel to "Add source".

Reviewer: ```suggestion
    3. To add directly to Media Encoder, click the "+" button in the Queue panel to "Add source".
```

3. Click the Format of the asset to access Export Settings.

![Open export settings](/wiki/images/open-export-settings.png)

> 4. Under the Export Settings, select a format of H.264 which results in a `.mp4` file which is appropriate for web. You can also optionally select the output location and name of the rendered asset.

Reviewer: ```suggestion
4. Under the Export Settings, select a format of H.264 which results in an MP4 file which is appropriate for web. You can also optionally select the output location and name of the rendered asset.
```
